bin/www

```diff
@@ -9,7 +9,6 @@
 /* eslint-disable global-require */
 
 function initialize() {
-  require('dotenv').config();
```

Comment:
Thanks for the help, @korymarks !

Please coordinate work with @JeremiahDotNet.

Also, our guidance from @naptowncode was that 12 factor and dotenv would coexist.

Reply:
It looks like 12-factor replaces the need to dotenv. The env file is loading correctly with 12-factor installed and without dotenv.